dash-renderer/src/components/error/CallbackGraph/CallbackGraphContainer.react.js

```diff
@@ -141,6 +141,26 @@ function flattenInputs(inArray, final) {
 // len('__dash_callback__.')
 const cbPrefixLen = 18;
 
+const layouts = {
+    'top-down': {
+        name: 'dagre',
+        padding: 10,
+        spacingFactor: 0.8
+    },
+    'left-right': {
+        name: 'dagre',
+        padding: 10,
+        nodeSep: 0,
+        rankSep: 80,
+        rankDir: 'LR'
+    },
+    force: {
+        name: 'cose',
+        padding: 10,
+        animate: false
+    }
```

Comment:
@nicolaskruchten I couldn't come up with a single layout that always seemed better than the others, so made a dropdown to choose among three. Another side benefit of this is if you drag nodes around to tweak the layout, and you want to get back to the original, you can change to a different algo and then back to the original.